doc/api/process.md

```diff
@@ -1356,6 +1356,47 @@ In custom builds from non-release versions of the source tree, only the
 `name` property may be present. The additional properties should not be
 relied upon to exist.
 
+## process.resourceUsage()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Object} containing resource usage measures for the current process
+
+Note: Windows only provides the following resource usage measures:
+
+- `userCpuTimeUsedSec`
+- `userCpuTimeUsedMs`
+- `systemCpuTimeUsedSec`
+- `systemCpuTimeUsedMs`
+- `blockInputOperations`
+- `blockOutputOperations`
+
+On Windows, all other values are `0`.
+
+Example return value:
+
+```js
+    { userCpuTimeUsedSec: 0,
+      userCpuTimeUsedMs: 159729,
+      systemCpuTimeUsedSec: 0,
+      systemCpuTimeUsedMs: 30597,
+      maxResidentSetSize: 20426752,
+      sharedMemSize: 0,
+      integralUnsharedDataSize: 0,
+      integralUnsharedStackSize: 0,
+      pageReclaims: 3002,
+      pageFaults: 2250,
+      swaps: 0,
+      blockInputOperations: 0,
+      blockOutputOperations: 0,
+      ipcMessagesSent: 55,
+      ipcMessagesReceived: 54,
+      signalsReceived: 0,
+      voluntaryContextSwitches: 92,
+      involuntaryContextSwitches: 374 }
+```
```

Comment:
Might be worthwhile to call out that older Linux kernels set some of these fields to zero, see http://stackoverflow.com/a/7205853.

I expect that `maxResidentSetSize` in particular will be used a lot but that isn't supported by <2.6.32 kernels (torvalds/linux@1f10206cf8e945220f7220a809d8bfc15c21f9a5 for the curious.)  Someone developing on a new kernel but deploying on 2.6.18 is in for a nasty surprise.